external/storm-kafka/src/jvm/storm/kafka/DynamicBrokersReader.java

```diff
@@ -149,4 +158,19 @@ private Broker getBrokerHost(byte[] contents) {
         }
     }
 
+    private void validateConfig(final Map conf) {
+        if(conf.get(Config.STORM_ZOOKEEPER_SESSION_TIMEOUT) == null) {
```

Comment:
These are all storm configs , even if user didn't pass these or set in storm.yaml there are defaults . So not sure checking for this necessary.